src/main/java/frc/robot/subsystems/drivetrain/BaseSwerveSubsystem.java

```diff
@@ -265,4 +265,11 @@ private Rotation2d getFieldHeading() {
             ? getGyroHeading().minus(angleOffset)
             : getRobotPosition().getRotation();
     }
+
+    public void lockNow(){
```

Comment:
Does this method work? I think currently `periodic()` would override these states every loop. Also small thing but this can be renamed `lock()` or something and periodic can also call this, if this method is even necessary (we can turn down the lock timeout if we need to lock faster on the charging station).

Reply:
If we get rid of this method then `BalancerCommand` can even easier be reverted back to just needing a `BaseDrivetrain`.